x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingLogo, EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { useCspBenchmarkIntegrations } from '../pages/benchmarks/use_csp_benchmark_integrations';
+import { useCISIntegrationPoliciesLink } from '../common/navigation/use_navigate_to_cis_integration_policies';
+import { NO_FINDINGS_STATUS_TEST_SUBJ } from './test_subjects';
+import { CloudPosturePage } from './cloud_posture_page';
+import { useCspSetupStatusApi } from '../common/api/use_setup_status_api';
+
+const NotDeployed = () => {
+  // using an existing hook to get agent id and package policy id
+  const benchmarks = useCspBenchmarkIntegrations({
+    name: '',
+    page: 1,
+    perPage: 1,
+    sortField: 'package_policy.name',
+    sortOrder: 'asc',
+  });
+
+  // the ids are not a must, but as long as we have them we can open the add agent flyout
+  const firstBenchmark = benchmarks.data?.items?.[0];
+  const integrationPoliciesLink = useCISIntegrationPoliciesLink({
+    addAgentToPolicyId: firstBenchmark?.agent_policy.id || '',
+    integration: firstBenchmark?.package_policy.id || '',
+  });
+
+  return (
+    <EuiEmptyPrompt
+      data-test-subj={NO_FINDINGS_STATUS_TEST_SUBJ.NO_AGENTS_DEPLOYED}
+      color="plain"
+      iconType="fleetApp"
+      title={<h2>{'No Agents Installed'}</h2>}
+      body={<p>To see findings, install an elastic agent on your cluster</p>}
```

Comment:
can we update this text from `To see findings, install an elastic agent on your cluster` to `To see findings, please finish the setup process by installing an elastic agent to your Kubernetes cluster.`

@JordanSh is this the prompt the user will see both on the findings page and dashboard?

Reply:
done